spec/src/main/asciidoc/microprofile-opentracing.asciidoc

```diff
@@ -192,6 +192,8 @@ Configuration of the skip pattern leverages MicroProfile Config specification.
 The skip pattern is specified as a string with key `mp.opentracing.server.skip-pattern` which has to be
 compliant with `java.util.regex.Pattern`. An example skip pattern might be `mp.opentracing.server.skip-pattern=/health|/metrics.*`
 
+The default value of the skip pattern excludes tracing for endpoints defined by MicroProfile Health, Config and OpenAPI specification.
```

Comment:
Should this be "Metrics" and not "Config"?

Reply:
lol I don't know what I wanted to exclude in config